styles/_helpers.scss

```diff
@@ -4,5 +4,5 @@
 }
 
 .is-hidden {
-  display: none;
+  display: none !important;
```

Comment:
I had to add the !important because for some reason, this class is being applied before the component styles. I don't know if you know how to fix this. I couldn't.